lib/mailer/sfMailer.class.php

```diff
@@ -274,31 +334,6 @@ public function sendNextImmediately()
     return $this;
   }
 
-  /**
-   * Sends the given message.
-   *
-   * @param Swift_Transport $transport         A transport instance
-   * @param string[]        &$failedRecipients An array of failures by-reference
-   *
-   * @return int|false The number of sent emails
-   */
-  public function send(Swift_Mime_Message $message, &$failedRecipients = null)
```

Comment:
Remove the type hint.

Reply:
Thanks, that was way to simple... ;-) I wasn't aware, that you can remove the typehint and be less specific while extending from another class.

Reply:
You rise me a doubt. But for sure the test have always right, and the result may depend on the PHP version.

Reply:
Yes it's working. On a new codebase I would not do prefer it that way, but on the legacy stuff it's sufficient to get the compat for swiftmailer 6 :)

Reply:
Yes, however, even on new codebase as the type constraint is respected with the call on the parent and the transport.

There is a goal to avoid using language version specific code as much as possible to avoid useless complexity.

But that's another story depending of the codebase. I give you me point of view.